pkg/apis/v1/nodeclaim.go

```diff
@@ -37,7 +37,6 @@ type NodeClaimSpec struct {
 	// Requirements are layered with GetLabels and applied to every node.
 	// +kubebuilder:validation:XValidation:message="requirements with operator 'In' must have a value defined",rule="self.all(x, x.operator == 'In' ? x.values.size() != 0 : true)"
 	// +kubebuilder:validation:XValidation:message="requirements operator 'Gt' or 'Lt' must have a single positive integer value",rule="self.all(x, (x.operator == 'Gt' || x.operator == 'Lt') ? (x.values.size() == 1 && int(x.values[0]) >= 0) : true)"
-	// +kubebuilder:validation:XValidation:message="requirements with 'minValues' must have at least that many values specified in the 'values' field",rule="self.all(x, (x.operator == 'In' && has(x.minValues)) ? x.values.size() >= x.minValues : true)"
```

Comment:
I'm not sure that we want to drop this requirement. It feels like if we are relaxing the minValues, we should also change the minValues requirement that we have associated with that NodeClaim -- that way we still maintain the requirement being passed onto the NodeClaim AND we don't have to remove the validation that we added here in CEL

Reply:
After discussing offline, we decided that we will allow the requirement for minValues to be relaxed so that we can have the NodeClaim have minValues requirements that match the number of values that will be passed through from the scheduler.

This will ensure that NodeClaims continue to be a standalone concept and make sense in isolation (outside the context of the scheduler) -- if we were to go the other way, we would have to write special code on the launch path that would create an exception for values not satisfying minValues if we know that the minValuesPolicy is set in the environment variables